kornia/geometry/conversions.py

```diff
@@ -679,7 +679,7 @@ def angle_axis_to_quaternion(angle_axis: torch.Tensor) -> torch.Tensor:
     quaternion[..., 0:1] += a0 * k
     quaternion[..., 1:2] += a1 * k
     quaternion[..., 2:3] += a2 * k
-    return torch.cat([w, quaternion], dim=-1)
+    return torch.cat([quaternion, w], dim=-1)
```

Comment:
if we go for [w, x, y, z] I guess we shouldn't modify this.

Reply:
Quite the feature creep...how do you propose to warn against backward compatibility?
Something like 
```python
def quaternion_to_rotation_matrix(quaternion: torch.Tensor, order: str = 'xyzw') -> torch.Tensor:
    if order not in ('wxyz', 'xyzw'):
        raise RuntimeError(f"Need order to be one of {'xyzw', 'wxyz'}")
   if order == 'xyzw':
        logger.warning(f'Order xyzw deprecated, please use wxyz')
```
?

Reply:
that's an option, yes

Reply:
Ok, I think it's ready for review again. Not pretty, but transitions never are.